crates/rpc-types/src/eth/account.rs

```diff
@@ -41,6 +41,20 @@ pub struct EIP1186AccountProofResponse {
     pub storage_proof: Vec<EIP1186StorageProof>,
 }
 
+/// Response for `eth_getAccount`
+#[derive(Copy, Clone, Debug, Default, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
```

Comment:
do we have an Account struct in alloy-consensus? if not i would prefer to push this down and ensure it has RLP encoding

Reply:
yeah we should either move this to consensus or eips, maybe eip1186 because more or less related